inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -907,7 +907,8 @@ def normalize_collaborations(obj, eng):
         else:
             collaboration_normalized_name = response[0].collaboration.value
         LOGGER.info(u"(%s) collaboration normalization: normalized: %s ==> %s", obj.id, collaboration['value'], collaboration_normalized_name)
-
+        if len(response.hits) > 1:
+            continue
```

Comment:
@MJedr it would have been good to add some logging here. The idea is that a curator will look at the logs and manually fix ambiguous cases. Sorry if it wasn't clear in the story.

Reply:
Ok, I'll add